be/src/vec/functions/array/function_array_enumerate.cpp

```diff
@@ -56,7 +56,7 @@ class FunctionArrayEnumerate : public IFunction {
     static constexpr auto name = "array_enumerate";
     static FunctionPtr create() { return std::make_shared<FunctionArrayEnumerate>(); }
     String get_name() const override { return name; }
-    bool use_default_implementation_for_nulls() const override { return false; }
+    bool use_default_implementation_for_nulls() const override { return true; }
```

Comment:
In the class `IFunction`, `use_default_implementation_for_nulls `has been `true`.

Reply:
Yes , can just delete this override func